tests/Custom/DesiredCapabilitiesTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
+
+use Behat\Mink\Driver\Selenium2Driver;
+use Behat\Mink\Tests\Driver\TestCase;
+
+class DesiredCapabilitiesTest extends TestCase
+{
+    public function testGetDesiredCapabilities()
+    {
+        $caps = array(
+            'browserName'       => 'firefox',
+            'version'           => '30',
+            'platform'          => 'ANY',
+            'browserVersion'    => '30',
+            'browser'           => 'firefox',
+            'name'              => 'Selenium2 Mink Driver Test',
+            'deviceOrientation' => 'portrait',
+            'deviceType'        => 'tablet',
+            'selenium-version'  => '2.45.0'
+        );
+
+        $driver = new Selenium2Driver('firefox', $caps);
+        $this->assertNotEmpty($driver->getDesiredCapabilities(), 'desiredCapabilities empty');
+        $this->assertTrue(gettype($driver->getDesiredCapabilities()) == 'array');
+        $this->assertEquals($driver->getDesiredCapabilities(), $caps);
```

Comment:
the expected value should be the first argument. Otherwise error messages are confusing on failure.